### PR TITLE
Update to v1.2.0

### PR DESCRIPTION
**Pull request by kneemer**

### CODE REVIEW

File: javadoc/reference/com/esri/android/geotrigger/GeotriggerBroadcastReceiver.ProcessLocationFailListener.html [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (JerrySievert) [inline javadoc/reference/com/esri/android/geotrigger/GeotriggerBroadcastReceiver.ProcessLocationFailListener.html]: wat?

File: javadoc/reference/com/esri/android/geotrigger/GeotriggerBroadcastReceiver.ProcessLocationFailListener.html [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
```
Comment (JerrySievert) [inline javadoc/reference/com/esri/android/geotrigger/GeotriggerBroadcastReceiver.ProcessLocationFailListener.html]: wat?

File: javadoc/reference/com/esri/android/geotrigger/GeotriggerBroadcastReceiver.ProcessLocationFailListener.html (hunk 1 above)
Comment (kneemer) [inline javadoc/reference/com/esri/android/geotrigger/GeotriggerBroadcastReceiver.ProcessLocationFailListener.html]: result of the javadoc generation, you will see that in all the generated html files.

**Comment by JerrySievert** [comment]
:+1:

**Comment by aaronpk** [comment]
:party:

File: javadoc/reference/com/esri/android/geotrigger/GeotriggerBroadcastReceiver.ProcessLocationFailListener.html (hunk 1 above)
Comment (doneill) [inline javadoc/reference/com/esri/android/geotrigger/GeotriggerBroadcastReceiver.ProcessLocationFailListener.html]: The output looks good, we have the same result in the android sdk.

Comment (JerrySievert) [inline javadoc/reference/com/esri/android/geotrigger/GeotriggerBroadcastReceiver.ProcessLocationFailListener.html]: frightening :)

**Pull request merged**